common/textconsole.h

```diff
@@ -78,6 +78,25 @@ void setErrorHandler(ErrorHandler handler);
  * @{
  */
 
+#define XXSTRING_X(s) #s
+#define XXSTRING(s) XXSTRING_X(s)
+
+#define scummvm_assert_always(cond, message) \
+	if (!(cond)) { \
+		error("%s - " XXSTRING(cond), message); \
```

Comment:
Here should be options to toggle a breakpoint (which would allow to attach a debugger before we crash the game)